__tests__/app.test.js

```diff
@@ -85,4 +83,35 @@ describe("GET /api/reviews/999999", () => {
         expect(body).toEqual({ msg: "not found" });
       });
   });
-});
\ No newline at end of file
+});
+
+describe("GET /api/users", () => {
+  test("200: should return an array of users", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        body.users.forEach((user) => {
```

Comment:
If you are using a forEach, you want to also test for the expected length of users array to make sure that it is running for all the objects which would be more robust.